frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue

```diff
@@ -0,0 +1,276 @@
+<template>
+  <section class="mini-composer">
+    <div
+      ref="configRef"
+      class="mini-composer__config-container"
+    >
+      <slot />
+    </div>
+    <div
+      ref="instructionsRef"
+      class="mini-composer__exercise-text"
+    />
+    <div class="mini-composer__gates">
+      <h1 class="mini-composer__gates__title">
+        Gates
+      </h1>
+      <GatesPool :available-gates="availableGates" />
+    </div>
+    <div class="mini-composer__circuit-section">
+      <h1 class="mini-composer__circuit-section__title">
+        Circuit
+      </h1>
+      <Circuit
+        :circuit-state="circuitState"
+        :auto-measure-gate="false"
+        :max-lines="circuitState.length"
+        :max-gates="MAX_GATES"
+      />
+    </div>
+    <div
+      ref="explanationRef"
+      class="mini-composer__explanation"
+    />
+    <div class="mini-composer__state-views">
+      <section class="mini-composer__state-views__view">
+        <h1 class="mini-composer__state-views__view__title">
+          Matrix
+        </h1>
+        <SymbolicNotation
+          class="mini-composer__state-views__view__area"
+          :tex="matrixTex"
+        />
+      </section>
+      <section class="mini-composer__state-views__view">
```

Comment:
@korgan00 - does this need a class? I don't see it being used. The title and area within the section _can_ be

`mini-composer__state-views__title`
`mini-composer__state-views__area`